amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -1490,11 +1490,8 @@ type ColonyFundsClaim @model {
 """
 Represents a native Colony Chain Funds Claim
 E.g., Token 0x0000...0000: ETH, xDAI, etc
-""" # This is really not elegant, but there's no other proper solution, as you can't
-type # return the Token @hadOne relationship from the lambda function
-# Note that we don't need token details for the values that we're fetching (since
-# it's implied), but it would have been nice not to have to basically repeat the type
-ColonyChainFundsClaim {
+""" # This is really not elegant, but there's no other proper solution, as you can't # return the Token @hadOne relationship from the lambda function # Note that we don't need token details for the values that we're fetching (since # it's implied), but it would have been nice not to have to basically repeat the type
```

Comment:
why have these separate lines all been combined into one?

if there a single line is correct should we remove these extraneous # symbols?

Reply:
Don't know, it's weird and potentially prettier related.